Source/gendung.h

```diff
@@ -248,5 +248,6 @@ void DRLG_Init_Globals();
 bool SkipThemeRoom(int x, int y);
 void InitLevels();
 void FindTransparencyValues(int i, int j, int x, int y, int d, uint8_t floorID);

```

Comment:
This is no longer called from outside gendung.h so no longer needs to be in the header.